app/assets/stylesheets/dashboard.scss

```diff
@@ -64,6 +74,14 @@ nav {
   padding-top: 20px;
 }
 
+.info-segment {
+  border-top:2px solid #EBEBEB;
```

Comment:
Color literals like `#EBEBEB` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#EBEBEB` should be written as `#ebebeb`<br>Colon after property should be followed by one space